first_5_steps/1_install_tt_buda.md

```diff
@@ -63,16 +63,37 @@ The [tt-firmware](https://github.com/tenstorrent/tt-firmware) file needs to be i
 
 ### Backend Compiler Dependencies
 
-Instructions to install the Tenstorrent backend compiler dependencies on a fresh install of Ubuntu Server 20.04.
+Instructions to install the Tenstorrent backend compiler dependencies on a fresh install of Ubuntu Server 20.04 or Ubuntu Server 22.04.
 
 *You may need to **append** each `apt-get` command with `sudo` if you do not have root permissions.*
 
+For both operating systems run the following commands:
+
+```bash
+apt update -y
+apt upgrade -y --no-install-recommends
+apt install -y build-essential curl libboost-all-dev libgl1-mesa-glx libgoogle-glog-dev libhdf5-serial-dev ruby software-properties-common libzmq3-dev 
+
+```
+
+For Ubuntu 20.04, add:
+```bash
+apt install -y clang-6.0 libyaml-cpp-dev libzmq3-dev python3.8-venv 
+```
+
+For Ubuntu 22.04, add:
+```bash
+apt install -y python3-venv clang wget
+wget http://mirrors.kernel.org/ubuntu/pool/main/y/yaml-cpp/libyaml-cpp-dev_0.6.2-4ubuntu1_amd64.deb
+wget http://mirrors.kernel.org/ubuntu/pool/main/y/yaml-cpp/libyaml-cpp0.6_0.6.2-4ubuntu1_amd64.deb
+dpkg -i libyaml-cpp-dev_0.6.2-4ubuntu1_amd64.deb libyaml-cpp0.6_0.6.2-4ubuntu1_amd64.deb
+rm libyaml-cpp-dev_0.6.2-4ubuntu1_amd64.deb libyaml-cpp0.6_0.6.2-4ubuntu1_amd64.deb
+```
+
+Then, for both operating systems, run:
+
 ```bash
-apt-get update -y
-apt-get upgrade -y --no-install-recommends
-apt-get install -y software-properties-common
-apt-get install -y python3.8-venv libboost-all-dev libgoogle-glog-dev libgl1-mesa-glx libyaml-cpp-dev ruby
-apt-get install -y build-essential clang-6.0 libhdf5-serial-dev libzmq3-dev
+apt install -y python3-pip python-is-python3 && rm -rf /var/lib/apt/lists/*
```

Comment:
If `python3-venv` works on both then you can move `apt install -y python3-pip python-is-python3` to the first section. I would just get rid of `rm -rf /var/lib/apt/lists/*` -- this way we simplify the steps